### PR TITLE
Add subsection for "backing up" YubiKey HMAC key. Concerns #465

### PR DESCRIPTION
**Pull request by ei-ke**
[NOTE]: # ( Describe your changes in detail, why is this change required? )
[NOTE]: # ( Explain large or complex code modifications. )
[NOTE]: # ( If it fixes an open issue, please add "Fixes #XXX" )
I've added a subsection to the _Database Operations_ section explaining how to generate the HMAC key and write it to a YubiKey so one has a) a backup and b) the option to have more than one YubiKey for using the optional Challenge-Response feature of KeePassXC.

PS: I just noticed that my editor removed spaces at the end of line. If this is an issue I'll take care of this in future.

## Screenshots
[TIP]:  # ( Do not include screenshots of your actual database! )

## Testing strategy
[NOTE]: # ( Please describe in detail how you tested your changes. )
[TIP]:  # ( We expect new code to be covered by unit tests and documented with doc blocks! )

## Type of change
[NOTE]: # ( Please remove all lines which don't apply. )
- ✅ Documentation (non-code change)

### CODE REVIEW

File: docs/topics/DatabaseOperations.adoc [hunk 1]
```diff
@@ -288,6 +288,8 @@ image::database_security.png[]
 .Database credentials
 image::database_security_credentials.png[]
 
+WARNING: Consider creating having a backup of your YubiKey. Please refer to <<Creating a YubiKey backup>>
```
Comment (louib) [inline docs/topics/DatabaseOperations.adoc:291]: @ei-ke `having` is not required here.

Comment (ei-ke) [reply docs/topics/DatabaseOperations.adoc]: Thanks, missed that one :)

**Comment by phoerious** [comment]
Please do not add spaces at the end of a line.

**Comment by ei-ke** [comment]
@phoerious Maybe I have expressed myself in a misleading way. Of course I'll not add them.
I just didn't know if it's annoying for reviewers when I was working on line 75-80 and my editor removed unnecessary spaces on the end ifnline 3, 50 and 223 :)

**Comment by phoerious** [comment]
They shouldn't be there. If newlines need to be escaped, it should be done with backslashes.

**Review by droidmonkey — APPROVED** [review]
I simplified the instructions and rebased on 2.6.4 branch

**Pull request merged**